src/exojax/atm/fastchem2_call.py

```diff
@@ -0,0 +1,169 @@
+#Several functions to simplify the integration of FastChem2 to Exojax, adopted from https://github.com/exoclime/FastChem/blob/master/python/
+# Dictionary of the name of chemical species available in the FastChem2
+# fc_dict=np.array(["P(bar)", "T(k)", "n_<tot>(cm-3)","n_g(cm-3)", "m(u)", "e-", "Al", "Ar", "C", "Ca", "Cl",
```

Comment:
you can put the data file in src/exojax/data/.
Do not forget add the data file name in [MANIFEST.in](https://github.com/HajimeKawahara/exojax/blob/master/MANIFEST.in).

Reply:
[This ](https://github.com/HajimeKawahara/exojax/blob/6bde59cffee96d813275c2464d13422a61f7bf53/src/exojax/spec/atomllapi.py#L291) is an example to read data file in MANIFEST.in.
Another examples is [here.](https://gist.github.com/HajimeKawahara/862428078a9ef412f6ea16fd0131ffaf)

Reply:
Thanks! I will add google style comments in the source file and move the data file